parking/src/models/Floor.java

```diff
@@ -0,0 +1,20 @@
+package models;
+
+import java.util.*;
+import constant.VehicleType;
+import models.ParkingLot;
+
+
+public class Floor {
+	
+	        
+	
+	public static void floorNumber(Vehicle vehicle){

```

Comment:
Wrong method, methods inside models should only set/return the properties declared within them.
There should be a property floorNumber, and the getter methods.